README.md

```diff
@@ -1 +1,92 @@
-# sound_manager
+# autonomous_driving_sound_manager
+
+## Overview
+Any autonomous driving vehicle have to notify its action to surrounding people.
+
+This node selects both a recorded voice and BGM depending on a state of whole system which combined the Autoware and a vehicle.
+
+This node plays sound in the following order of priority.
+1. Alert people around the ego vehicle once before the ego vehicle departs. The ego vehicle will not depart until the alert is complete.
```

Comment:
What do these "Alert"s play by ? voice or BGM or anything others. Add relationships between them.

Reply:
This issue will be fixed by https://github.com/eve-autonomy/ad_sound_manager/pull/3